module/observationSubmissions/helper.js

```diff
@@ -118,6 +118,7 @@ module.exports = class ObservationSubmissionsHelper {
               solutionDocument = solutionDocument[0];
               observationSubmissionsDocument['solutionInfo'] = solutionDocument;
 
+              if(observationSubmissionsDocument.programId){
```

Comment:
@borkarsaish65 please format the code correct the indendation

Reply:
done